src/edu/csus/ecs/pc2/ui/ISelectedListsSetter.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui;
+
+import java.util.List;
+
+/**
+ * Set/Return selected valuea or indexes.
```

Comment:
This comment is misleading; there is no method defined in this interface that "Returns" anything.  Please provide a more accurate description (preferably multiple lines) explaining what the interface is for/how it works.   
Also please correct the typo in the above line (what is "valuea" supposed to be?)